.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,43 @@
+name: Changelog
+
+on:
+  pull_request:
+    types: [ "opened", "synchronize", "labeled", "unlabeled" ]
+
+jobs:
+  require-changelog:
+    name: Is changelog required?
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Retrives changed files in CHANGELOG/unreleased/**/*.yaml
+        id: changelog-check
+        uses: tj-actions/changed-files@v37
```

Comment:
please make sure external actions are pinned to commit sha (and version as comment), like
```
uses: tj-actions/changed-files@aaaaaaaaaaaaaaa # v37
````

Reply:
Done